db/setup.sql

```diff
@@ -0,0 +1,20 @@
+-- CREATE TYPE org_status IF NOT EXISTS AS ENUM ('initial', 'allocated', 'connected');
+
+CREATE TABLE IF NOT EXISTS orgs (
```

Comment:
Can we call this table `deployments`? end-users will also use this system

Reply:
We can, but is it necessary? End-users won't interact with this, as we discussed. Any further configuration would be through Radicle network itself.

Reply:
Well it's necessary to make the code understandable.. we could call it banana otherwise?

Reply:
Fixed https://github.com/radicle-dev/radicle-cloud/pull/1/commits/a3bbef7206442aae4e3eca9af89c54b7e0edc684